iroh-relay/src/server.rs

```diff
@@ -886,7 +907,7 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn test_relay_clients_both_derp() {
+    async fn test_relay_clients_both_relay() -> Result<()> {
```

Comment:
I'd recommend `TestResult` instead.
The difference is that `TestResult` immediately panics on every `?`.
This helps locate the actual line that the error failed on quickly.
```suggestion
    async fn test_relay_clients_both_relay() -> TestResult {
```
(Same with the other tests below)

Reply:
we can do this later, but we haven't been using asfaik in here

Reply:
Done, but /cc @dignifiedquire as he might have written this.  (not actually sure)

Reply:
I don't mind